upgrade_test.py

```diff
@@ -141,7 +144,7 @@ class UpgradeTest(FillDatabaseData, loader_utils.LoaderUtilsMixin):
 
     def setUp(self):
         super().setUp()
-
+        self.do_truncates = True
```

Comment:
Let's do that at configuration level

It's something useful to be able to control on all upgrade tests

Reply:
Done